std/algorithm/iteration.d

```diff
@@ -1492,6 +1492,24 @@ if (isInputRange!R)
         tuple(4, 3u), tuple(5, 1u) ][]));
 }
 
+/**
+ * Using group, an associative array can be easily generated with the count of each
+ * unique element in the range.
+ */
+@safe unittest
+{
+    import std.algorithm.sorting : sort;
+    import std.array : assocArray;
+
+    uint[string] result;
+    auto range = ["a", "b", "a", "c", "b", "c", "c", "d", "e"];
+    result = range.sort!("a < b")
```

Comment:
Unfortunately it can't be omitted, because then the compiler assumes you're using `array.sort` rather than `std.algorithm.sort()`.

And for some reason, unlike Phobos, DMD does not stick to actually deprecating things when it says it will. `array.sort` has been a warning for almost three years now.